src/get-started/test-service.md

```diff
@@ -127,12 +129,56 @@ node, and a single `CurrencyService`. `TestKit` supports
 testing several services at once, as well as more complex network
 configurations, but this functionality is not needed in our case.
 
+### Other Helper Functions
+
+We need a way to obtain [the service schema](create-service.md#create-schema)
+from the testkit. The testkit allows to obtain a snapshot of the blockchain
+state, using which we construct the schema:
+
+```rust
+fn get_schema<'a>(
+    snapshot: &'a dyn Snapshot,
+) -> CurrencySchema<impl Access + 'a> {
+    CurrencySchema::new(snapshot.for_service(INSTANCE_NAME).unwrap())
+}
+```
+
+Here, `for_service` method of the `SnapshotExt` extension trait
+provides access to the data of a specific service. Since we do not care
+about the exact access type, we use generic `impl Access + 'a` as the
+type parameter of the resulting schema.
+
+!!! note
+    Use of lifetime `'a` indicates that the access type cannot outlive
+    the snapshot it is created from.
+
+Finally, we define getters to obtain wallet info from the blockchain
+state:
+
+```rust
+/// Returns the wallet identified by the given public key
+/// or `None` such wallet doesn't exist.
```

Comment:
Is *if* skipped ? Or **if** the wallet doesn't exist ?

Reply:
Yeah, there is a typo in the actual comment in the code. I'll fix it.